cmake/test_util.cmake

```diff
@@ -0,0 +1,29 @@
+################################################################################
+# Utility functions for creating tests
+
+################################################################################
+# build and add a standard test
+
+function(create_standard_test test_name test_src)
```

Comment:
If you wanted, you could omit the `test_src` argument and use CMake's variable arguments to allow a space-separate set of source files, just like the native CMake functions.

Reply:
See `ARGN` in the CMake documentation if you're curious.